docs/get-started/sign-up.md

```diff
@@ -62,7 +57,41 @@ After 30 days, your Sumo Logic **Trial account** will convert to a **Free accoun
 <sup>3</sup> Forwards your logs to an S3 bucket after being collected and analyzed in Sumo Logic.<br/>
 <sup>4</sup> Notifies you in real-time when errors occur on your systems.
 
-For more information, see [Sumo Logic Account Usage](/docs/manage/manage-subscription).
+ To unlock premium features and scale your Sumo Logic usage as your needs grow, upgrade to a paid plan. For more information, see [Sumo Logic Pricing](https://www.sumologic.com/pricing) and [Manage Subscription](/docs/manage/manage-subscription).
+
+### Upgrade to a Flex plan
+
+Using our service checkout, you can quickly and easily sign up for our Flex Essentials plan with a credit card in just minutes.  
+
+The Flex Essentials plan is ideal for:  
```

Comment:
Is flex essentials plan referring to this - https://www.sumologic.com/pricing/?igaag=144244772630&igaat=&igacm=18138484988&igacr=703522989243&igakw=sumo%20logic%20pricing&igamt=e&igant=g

Reply:
I took it from the FAQ (_We are introducing a Self-Checkout Page to allow users to purchase Sumo Logic plans directly through the product interface. Customers who are in free and trial plan type will be able to upgrade to **flex essentials** easily_.)